remote/supervisord.xpra.conf

```diff
@@ -2,7 +2,7 @@
 # SPDX-License-Identifier: Apache-2.0
 
 [program:xpra]
-command=xpra start :10 --start=/home/techuser/.config/openbox/autostart --start-env=GTK_IM_MODULE=ibus --attach=yes --daemon=no --bind-tcp=0.0.0.0:10001 --min-quality=70
+command=xpra start-desktop :10 --start=/usr/local/bin/openbox-launcher.sh --attach=yes --daemon=no --bind-tcp=0.0.0.0:10001 --min-quality=70
```

Comment:
Since openbox-launcher is now used as startup command, the favicon and title of the Tab in Chrome relate to "Openbox". It would be nice to still see the window title - or at least the text "Capella".

Reply:
I tried this, but it doesn't seem like there's an easy win here. We could try to hard-code the title "Capella" in the HTML/JS files somehow, but it gets set dynamically after connecting, so this doesn't exactly sound trivial.